R/elasticsearch_eda_funs.R

```diff
@@ -32,6 +32,7 @@
 #'                      , end_date = "now"
 #'                      , time_field = "dateTime")
 #' }
+#' @references \href{https://github.com/UptakeOpenSource/uptasticsearch/pull/69}{get_counts will be deprecated soon}
```

Comment:
Perhaps change this to `@note`? That may make more sense than "References"

Reply:
good call